components/serverless/config/crd/bases/serverless.kyma-project.io_functions.yaml

```diff
@@ -216,6 +216,9 @@ spec:
                         description: Defines the name of the predefined set of values
                           of the resource. Can't be used together with **Resources**.
                         type: string
+                        x-kubernetes-validations:
+                        - message: 'Invalid profile, please use one of: [''XS'',''S'',''M'',''L'',''XL'']'
+                          rule: (self in ['XS','S','M','L','XL'])
```

Comment:
How it correspond with:
https://github.com/kyma-project/serverless/blob/main/components/serverless/hack/webhook_config.yaml
If I know for build we have presets "slow", "fast" ...

Reply:
Good catch

Reply:
Good catch, should be fixed